GoogleForCreators/web-stories-wp
# Document Panel: Add Taxonomies section

## Context

Should add a new tab to the document panel that is behind the new taxonomy feature flag. When that flag is on should get a new tab for Categories and Tags.

## Summary

For now, just added the title "Categories & Tags" and left the section empty.
Demoed in storybook and initial test added.

## User-facing changes



## Testing Instructions




This PR can be tested by following these steps:

1. Under experiments ensure that the feature flag for "taxonomies support" is enabled.
2. Navigate to edit a story, then select "Document"
3. Verify that at the end of the list the "Categories and Tags" collapsible item is there. 
4. When you click on it, the component with open, however nothing is inside yet. The title font weight will change matching the styles of the others. 

### Does this PR have a security-related impact?
No



### Does this PR change what data or activity we track or use?
No



### Does this PR have a legal-related impact?
No



## Checklist




---



Fixes #8835
